ogr/abstract.py

```diff
@@ -48,12 +65,44 @@ def __init__(
         self.created = created
         self.edited = edited
 
+    def __str__(self) -> str:
+        comment = f"{self.comment[:10]}..." if self.comment is not None else "None"
+        return (
+            f"PRComment("
+            f"comment='{comment}', "
+            f"author='{self.author}', "
+            f"created='{self.created}', "
+            f"edited='{self.edited}')"
+        )
+
 
 class CommitStatus:
```

Comment:
I do not want to break backwards compatibility, but what about using `flag` for this thing?

Reply:
what flag?

Reply:
`CommitFlag`/`PullRequestFlag`...

Reply:
Sounds good. You can even keep this class, mark it as deprecated and use inheritance to introduce the Flag classes.

Reply:
I've created a new issue so we can merge this.

https://github.com/packit-service/ogr/issues/73